app/views/Notification.js

```diff
@@ -159,21 +192,44 @@ class NotificationScreen extends Component {
               </View>
 
               <ScrollView contentContainerStyle={styles.contentContainer}>
-                {this.state.data.map(data => (
-                  <View key={data.x} style={styles.notificationView}>
-                    <Text
-                      style={[
-                        styles.notificationsText,
-                        data.y > 80
-                          ? styles.notificationsTextRed
-                          : data.y > 50
-                          ? styles.notificationsTextOrange
-                          : null,
-                      ]}>
-                      {'Day ' + data.x + ': ' + data.y + ' intersections'}
-                    </Text>
-                  </View>
-                ))}
+                {this.state.data.map(data =>
+                  data.y === 0 ? (
+                    data.x == 13 && !hasExposure ? (
+                      <Text style={styles.noExposure}>
+                        No known exposures to COVID during the last two weeks.
+                      </Text>
+                    ) : (
+                      <Text style={{ height: 0 }}></Text>
+                    )
+                  ) : (
+                    <View key={data.x} style={styles.notificationView}>
+                      <Text
+                        style={[
+                          styles.notificationsText,
+                          data.y > 80
```

Comment:
Make these thresholds into consts, or an enum. They're used in multiple places.

`const THRESHOLD_HIGH = 80`

or 

```js
const Threshold = {
  HIGH: 80,
  MEDIUM: ...
}
```

Reply:
You are picky in the right ways, my friend.

Reply:
I review a lot of code on the daily, heh.